configJson.js

```diff
@@ -0,0 +1,24 @@
+const configBuilder = require('openfin-config-builder');
```

Comment:
File name is very generic - maybe something like `updateOpenFinConfigForDeployment.js`?